src/main/java/DatabaseInteractin/ExecuteQuerys.java

```diff
@@ -0,0 +1,42 @@
+package DatabaseInteractin;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Scanner;
+
+public class ExecuteQuerys {
+  DataBaseInteracting dataBaseInteracting = new DataBaseInteracting();
+  Scanner sc = new Scanner(System.in);
+
+  public ExecuteQuerys() throws SQLException {
+  }
+
+  public List<Product> listReturn() throws SQLException {
+    List<Product> list;
+    try (PreparedStatement preparedStatement = dataBaseInteracting.createPrepareStatement();
```

Comment:
`createPreparedStatement` is not a good name too, it doesn't show any meaning (prepared statement may be anything)

Reply:
Done.